client/src/services/auth.service.js

```diff
@@ -59,9 +59,11 @@ async function logout() {
     await axios.post("/api/v1/users/logout", { refreshToken });
   } catch {
     // Ignore error
+  } finally {
+    localStorage.removeItem(LOCAL_STORAGE_KEY);
+    queryClient.clear();
+    window.location.replace("/login");
```

Comment:
If doing this here, we should 
- make it clear in the function name that we reload the document.
- remove the call to `location.replace` after `AuthService.logout()` in `Profile.js`
- make sure that no code is executed after `AuthService.logout()` in the whole app

Reply:
You're right is not the place to do that, I'll change that !